AmplifyPlugins/Core/AWSPluginsCore/Auth/AWSAuthService.swift

```diff
@@ -84,6 +106,27 @@ public class AWSAuthService: AWSAuthServiceBehavior {
         return .success(convertedDictionary)
     }
 
+    /// Retrieves the Cognito token from the AuthCognitoTokensProvider
+    /// - Parameter completion: Completion handler defined for the input `Result<String, AuthError>`
+    public func getToken(completion: @escaping (Result<String, AuthError>) -> Void) {
```

Comment:
Should we rename this as `getUserPoolAccessToken`? `getToken` is too generic; cognito User Pool returns accessToken, idToken and refreshToken.

Reply:
+1

Reply:
I'll defer to your Auth knowledge for proper naming here. Since we're renaming, my suggestion would be to drop the `get` prefix. So the new methods would be `userPoolAccessToken(…)` and `identityID(…)` (notice the capitalization change from `Id` to `ID`).

Thoughts?

Reply:
I recant dropping the `get` prefix.
Method names are now `getUserPoolAccessToken(completion:)` and `getIdentityID(completion:)`.
Addressed in 7f71335.
Tests also added in eb1fe4c.